reactor-netty-http/src/main/java/reactor/netty/http/server/NonSslRedirectHandler.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2011-Present VMware, Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.netty.http.server;
+
+import io.netty.channel.ChannelFutureListener;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelInboundHandlerAdapter;
+import io.netty.handler.codec.http.DefaultFullHttpResponse;
+import io.netty.handler.codec.http.HttpRequest;
+import io.netty.handler.codec.http.HttpResponse;
+import io.netty.handler.codec.http.HttpResponseStatus;
+
+import javax.annotation.Nullable;
+
+import static io.netty.handler.codec.http.HttpHeaderNames.HOST;
+import static io.netty.handler.codec.http.HttpHeaderNames.LOCATION;
+
+/**
+ * @author James Chen
```

Comment:
Add `@since 1.0.5`

Can you add 1-2 sentences that describe the handler. Add also that this handler will not propagate the incoming message when a response is completed.

Reply:
Done.

> The handler sends an HTTP response with a status code of 308 and a
> location header consisting of the host header of the request and
> HTTPS protocol to the incoming HTTP requests.
> <p>
> The handler is applicable only for HTTP/1.x and will not propagate
> for incoming HTTP requests